custom_components/sat/climate.py

```
@@ -85,6 +85,9 @@
     def __init__(self, coordinator: SatDataUpdateCoordinator, config_entry: ConfigEntry, unit: str):
         super().__init__(coordinator, config_entry)
 
+        # Setup some public variables
+        self.thermostat = config_entry.data.get(CONF_THERMOSTAT)
```

Ruff (0.8.2)

`CONF_THERMOSTAT` may be undefined, or defined from star imports

(F405)